model/Database.php

```diff
@@ -1045,4 +1044,39 @@ function removeFile($itemId)
         $statement = $this->_dbh->prepare($sql);
         $statement->execute([$itemId]);
     }
+
+    /**
+     * Gets the ItemTutorYear data
+     * @param string $currentYear the current year
+     * @param int $tutorId tutor year id
+     * @return array Array of data in ItemTutorYear table, TutorYear and Tutor
+     * @author Laxmi Kandel
+     */
+    function getItemTutor($currentYear, $tutorId)
+    {
+        $sql = "select * from tutors.ItemTutorYear
+                inner join tutors.TutorYear on ItemTutorYear.tutorYear_id = TutorYear.tutorYear_id
+                inner join tutors.Tutor on TutorYear.user_id = Tutor.user_id
+                where ItemTutorYear.tutorYear_id = ?
+                and Tutor.user_id = ?";
+        $statement = $this->_dbh->prepare($sql);
+        $statement->execute([$currentYear, $tutorId]);
+        return $statement->fetchAll(PDO::FETCH_ASSOC);
+    }
+
+    /**
+     * Get the yearId of the given tutor
+     * @param int $userId tutor's id
+     * @return mixed array of data in TutorYear and Tutor table
```

Comment:
if this function is to grab only the yearID of the given tutor, should we be returning more than the yearID?

Reply:
No, but I was thinking if we need other stuff we don't have to write another function again. But we are almost done with the functionality so, I can just change that to just grab what I need. I will update it soon! Thanks